snowplow-tracker/src/androidTest/java/com/snowplowanalytics/snowplow/event/AbstractEventTest.kt

```diff
@@ -24,14 +24,14 @@ class AbstractEventTest {
     @Test
     fun testAddsEntitiesUsingAllAPIs() {
         val event = ScreenView("screen")
+        val entity = SelfDescribingJson("schema3", "data3")
 
         event.entities.add(SelfDescribingJson("schema1", "data1"))
-        event.customContexts.add(SelfDescribingJson("schema2", "data2"))
-        event.entities(listOf(SelfDescribingJson("schema3", "data3")))
-        event.contexts(listOf(SelfDescribingJson("schema4", "data4")))
+        event.contexts(listOf(SelfDescribingJson("schema2", "data2")))
```

Comment:
We are not testing the outcome of these two calls – should we have asserts here? Better to swap their order so that we test that the `add()` adds to the existing entities.